ucte/ucte-network/src/test/java/com/powsybl/ucte/network/UcteElementIdTest.java

```diff
@@ -40,8 +40,6 @@ public void testHashCode() {
         UcteElementId id2 = new UcteElementId(node2, node1, '3');
         UcteElementId id3 = new UcteElementId(node1, node2, '2');
 
-        assertFalse(id1.equals(null));
```

Comment:
You delete it without replacing it by assertNotNull, is it what you wanted to do?

Reply:
Do you really think this test is interesting?